src/main/features/response/form/models/howMuchPaid.ts

```diff
@@ -0,0 +1,55 @@
+import { Serializable } from 'models/serializable'
+import { IsDefined, IsPositive, MaxLength } from 'class-validator'
+import { IsPastDate } from 'forms/validation/validators/datePastConstraint'
+import { LocalDate } from 'forms/models/localDate'
+import { IsNotBlank } from 'app/forms/validation/validators/isBlank'
+
+export class ValidationErrors {
+  static readonly NOT_OWE_FULL_AMOUNT_REQUIRED: string = 'You haven’t explained why you don’t owe the full amount'
+  static readonly REASON_NOT_OWE_MONEY_TOO_LONG: string = 'Enter reason no longer than $constraint1 characters'
+  static readonly VALID_AMOUNT_REQUIRED: string = 'Enter a valid amount paid'
+  static readonly AMOUNT_REQUIRED: string = 'Enter an amount'
+  static readonly DATE_REQUIRED: string = 'Enter a date'
+  static readonly VALID_PAST_DATE: string = 'Enter a valid date in the past'
+}
+
+export class HowMuchPaid implements Serializable<HowMuchPaid> {
+
+  @IsDefined({ message: ValidationErrors.AMOUNT_REQUIRED })
+  @IsPositive({ message: ValidationErrors.VALID_AMOUNT_REQUIRED })
+  amount?: number
+  @IsDefined({ message: ValidationErrors.DATE_REQUIRED })
+  @IsPastDate({ message: ValidationErrors.VALID_PAST_DATE })
+  date?: LocalDate
+  @IsDefined({ message: ValidationErrors.NOT_OWE_FULL_AMOUNT_REQUIRED })
+  @IsNotBlank({ message: ValidationErrors.NOT_OWE_FULL_AMOUNT_REQUIRED })
+  @MaxLength(99000, { message: ValidationErrors.REASON_NOT_OWE_MONEY_TOO_LONG })
+  text?: string
+  constructor (amount?: number, date?: LocalDate, text?: string) {
```

Comment:
add line break above here

Reply:
For some strange reason it doesn't allow me to do this in intelliJ

Reply:
you mean it's displayed as an error or ENTER doesn't work?

Reply:
I mean when I try to add a line break by pressing enter an error is shown in intelliJ

Reply:
you need to get rid of the spaces that intellij adds

Reply:
Sorted it out